Celeste.Mod.mm/Mod/Everest/Everest.Loader.cs

```diff
@@ -487,9 +487,13 @@ public static class Loader {
                     using (ZipFile zip = new ZipFile(meta.PathArchive)) {
                         foreach (ZipEntry entry in zip.Entries) {
                             string entryName = entry.FileName.Replace('\\', '/');
-                            if (entryName == meta.DLL) {
-                                using (MemoryStream stream = entry.ExtractStream())
+
+                            if (Path.GetExtension(entryName) == ".dll") {
+                                using MemoryStream stream = entry.ExtractStream();
+                                if (entryName == meta.DLL)
                                     asm = Relinker.GetRelinkedAssembly(meta, Path.GetFileNameWithoutExtension(meta.DLL), stream);
+                                else // immediately load any other libraries to avoid reading zip files over and over again later
+                                    Relinker.GetRelinkedAssembly(meta, Path.GetFileNameWithoutExtension(entryName), stream);
```

Comment:
I think only dll files in the same directory as meta.DLL need to be loaded

Reply:
Yeah that's more consistent with how it used to work, just implemented this